### DISCUSSION TITLE
Add more of the discussion features to mobile

### DISCUSSION THREAD

**Discussion by ririinsta** [community/community#120546] [Mobile] [answered]
This isn't really an issue more of an idea. I like to manage the discussions for some of my projects from my phone while I'm busy doing other things. Currently there is no way to mark things as resolved or off topic on mobile. Is there any plans to add this?

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.158.0 (128225528)
Device: iPhone 15 (iOS 17.4.1)
</details>

**Comment by aconsuegra** [comment]
Hi @ririinsta, thanks for getting in touch! It should be possible to mark comments as answers on Mobile, do you have more details or could you maybe record a video about the behavior that you are experiencing?

Also, when you mention "to mark things as off topic", do you mean adding a label or changing the category?

**Comment by ririinsta** [comment]
I am sorry I did not specifically say what I was requesting/suggesting. Specifically I am suggesting the hide feature. I have a few screenshots as an example.

This one showing the menu in GitHub Mobile but present in an issue.
![image](https://github.com/community/community/assets/87190112/eb76f93d-30bd-40e5-9d98-0cd31e6f0096)

This one showing the same menu on a thread but in a discussion. 
![image](https://github.com/community/community/assets/87190112/287c531d-396f-4534-be8d-94c631854dcc)

These two showing the feature present in threads on the website version.
![image](https://github.com/community/community/assets/87190112/750dd301-811a-4bf0-b587-fefbfbc99fdf)
![image](https://github.com/community/community/assets/87190112/763a64fb-1aa0-4611-b03e-8c2ca8dd9eec)

**Reply by aconsuegra** [reply, marked as answer]
Thanks for the extra context! I'm going to create an issue to track this internally, and hopefully we can include the feature in one of our upcoming releases.

**Reply by ririinsta** [reply]
Ok thank you!
